zips/zip-0234.md

```diff
@@ -129,10 +129,10 @@ $\mathsf{DEPLOYMENT\_BLOCK\_HEIGHT} = \mathsf{TBD}$
 The block height will be chosen by the following criteria:
 
 - It is after NU7 activation height.
-- It is calculated to be the lowest height after the second halving at
-  which the NSM issuance would be less than the current BTC-style issuance
-  _neglecting_ any burnt ZEC (i.e. assuming the amount of ZEC burnt is
-  exactly 0).
+- It is calculated to be the lowest height after the second halving at which
+  the NSM issuance would be less than the current BTC-style issuance _neglecting_
+  any ZEC removed from circulation (i.e. assuming the amount of ZEC removed from
+  circulation is exactly 0).
 
 This selection is intended to achieve Key Objective 6 while still being at
```

Comment:
This paragraph, and the one following it, should both be in a `### Rationale for parameters` subsection; they are not part of the Specification. See [this part of ZIP 0](https://zips.z.cash/zip-0000#zip-format-and-structure) for more guidance.